src/Symfony/Component/Intl/Resources/data/timezones/bs.json

```diff
@@ -438,9 +438,6 @@
     },
     "Meta": {
         "GmtFormat": "GMT %s",
-        "HourFormat": [
-            "+%02d:%02d",
-            " -%02d:%02d"
-        ]
+        "HourFormatNeg": " -%02d:%02d"
```

Comment:
missing HourFormatPos?

Reply:
it'll be inherited from root.json (`"HourFormatPos": "+%02d:%02d"`), which is the same format as `bs` locale (`+HH:mm`)

i've no idea why the negative format has a leading space :man_shrugging: (upstream data), these cases shoud be confirmed by native speakers IMHO.

this basically shows the de-duplication works :D